service-workerBackupForURL.js

```diff
@@ -0,0 +1,51 @@
+const FILES_TO_CACHE = [
+    './css/modules/test.css',
+  ];
+
+const CACHE_NAME = 'pages-cache-v1';
+
+self.addEventListener('install', (event) => {
+    console.log('Inside the install handler:', event);
+    // event.waitUntil(
+    //     caches.open(CACHE_NAME)
+    //     .then(cache => {
+    //       console.log('[ServiceWorker] Pre-caching offline page');
```

Comment:
Codacy found an issue: [Expected indentation of 2 spaces but found 4.](https://app.codacy.com/gh/congrea/virtualclass/pullRequest?prid=5740126)